wled00/FX.cpp

```diff
@@ -4954,11 +4957,11 @@ uint16_t mode_2DColoredBursts() {              // By: ldirko   https://editor.so
       SEGMENT.setPixelColorXY(y1, y2, DARKSLATEGRAY);
     }
   }
-  if (SEGMENT.custom3) SEGMENT.blur(SEGMENT.custom3/2);
+  SEGMENT.blur(SEGMENT.custom3<<1, SEGMENT.check2);
```

Comment:
Contrary to the original this now doubles blurring instead of halving it.
This will cause *different looking* effect on existing presets. Please consider backwards compatibility.